example/server/config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"os"
+	"strings"
+)
+
+const (
+	// default port for the http server to run
+	DefaultIssuerPort = "9998"
+)
+
+type Config struct {
+	Port         string
+	RedirectURIs []string
+	UsersFile    string
+	Issuer       string
+}
+
+// FromEnvVars loads configuration parameters from environment variables.
+// If there is no such variable defined, then use default values.
+func FromEnvVars(defaults *Config) *Config {
+	if defaults == nil {
+		defaults = &Config{}
```

Comment:
Shouldn't `DefaultIssuerPort` be set to the Config here?

Reply:
That works as planned - if the defaults is nil, then user didn't want to have any defaults and expects everything to be set in the env vars.